src/Core/Utils.fs

```diff
@@ -194,18 +212,15 @@ module Markdown =
         comment
         |> String.replace "&lt;" "<"
         |> String.replace "&gt;" ">"
-        |> (fun v -> v.Split '\n')
-        |> Array.filter(String.IsNullOrWhiteSpace>>not)
-        |> Array.mapi (fun i line ->
-            let v =
-                if i = 0 && not (String.IsNullOrWhiteSpace line)
-                then "\n" + line.Trim()
-                else line.Trim()
-            replaceXml v
-            )
+        |> String.replace "&quot;" "\""
```

Comment:
It may be good idea to put those replacements into separate list, and fold over it - it will make it bit easier to add next replacements in the future if we're still missing something

Reply:
done